src/main/java/Duke/common/Response.java

```diff
@@ -0,0 +1,37 @@
+package duke.common;
+
+/**
+ * Stores all generic string responses back to user.
+ *
+ * @author Oh Jun Ming
+ * @version 1.0
+ */
+public enum Response {
+    GREET("Hello! I'm Duke\nWhat can I do for you?\n"),
+    EXIT("Bye. Hope to see you again soon!\n"),
+    ADD("Got it. I 've added this task:\n"),
+    DELETE("Noted. I've removed this task:\n"),
+    DONE("Nice! I 've marked this task as done\n"),
+    LIST("Here are the tasks in your list:\n");
+
+    private final String s;
```

Comment:
Do try to give a meaningful names to class variables. As for this case, a suggestion would be to use "string" or "response" instead of "s".